pkg/volume/plugins.go

```diff
@@ -118,6 +118,10 @@ type VolumePlugin interface {
 	// const.
 	CanSupport(spec *Spec) bool
 
+	// IsMigrated tests whether the plugin is migrated seamlessly to a CSI
+	// Driver
+	IsMigrated() bool
```

Comment:
Will it be better to make this a bit clearer that it is `Migrated` in the context of CSI? Say `IsPluginMigratedToCsi`?

Reply:
Just realized the comment was enhanced to clarify this in response to Jan's comment earlier. I guess comment is good for now.

Reply:
I like this better ^